### ISSUE TITLE
WTSAPI32.dll does not run MCC.

### ISSUE THREAD

**Issue by kiminru** [WinterSquire/AlphaRing#35]
I installed Microsoft Visual C++ Redistributable and put the file in Halo The Master Chief Collection\MCC\Binaries\Win64
but it won't run. I tried with EAC turned off, but it still doesn't work. 
I tried deleting the WTSAPI32.dll file and running MCC and it works fine. I am using Windows 11 in Korean.
![스크린샷 2024-06-24 123254](https://github.com/WinterSquire/AlphaRing/assets/84241069/cd257892-8c76-4c3c-84bc-095727183ade)

**Comment by WinterSquire** [comment]
Did you try to [repair](https://aka.ms/vs/17/release/vc_redist.x64.exe) Microsoft Visual C++ Redistributable?

**Comment by kiminru** [comment]
> Did you try to [repair](https://aka.ms/vs/17/release/vc_redist.x64.exe) Microsoft Visual C++ Redistributable?

Yes, I tried to recover. I've uninstalled and reinstalled all Visual c++, but the result is the same.

**Comment by WinterSquire** [comment]
What is your system locale? Did you use `English (United States)`? Can you test the dev [build](https://ci.appveyor.com/project/WinterSquire/alpharing/build/artifacts)?

**Comment by kiminru** [comment]
![스크린샷 2024-07-27 225320](https://github.com/user-attachments/assets/1ad9dd1f-071d-4306-981b-21f5919dd0c0)
![스크린샷 2024-07-27 225551](https://github.com/user-attachments/assets/0b889f1f-8cf0-45b7-ba12-aef46e4a7b1c)

Sorry for the Korean display, I changed the system locale you mentioned to English (United States), 
and I'll also show you that I have visual c++ 2015-2022 (x64) installed.
As expected, the game doesn't start at all. To be more precise, the game doesn't turn on, it just crashes again. 
I don't know what the problem is, I've tried reinstalling visual c++ 2015-2022 (x64), I've tried everything, 
but it's still a problem. It runs fine on my steamdeck though :) 
Of course, my desktop is still the problem.
